core/amber/src/main/scala/edu/uci/ics/amber/engine/common/client/ClientActor.scala

```diff
@@ -78,7 +78,7 @@ private[client] class ClientActor extends Actor with AmberLogging {
           mId,
           _ @WorkflowFIFOMessage(_, _, _ @ReturnInvocation(originalCommandID, controlReturn))
         ) =>
-      sender ! NetworkAck(mId)
+      sender ! NetworkAck(mId, 0) // client does not have queued credits
```

Comment:
align them, so that we define a getQueuedCredit function and return 0? The function can be inline.